AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -866,7 +867,26 @@ - (void)processLogBufferAfterCrash {
     if ([[fileURL pathExtension] isEqualToString:kMSLogBufferFileExtension]) {
       NSData *serializedLog = [NSData dataWithContentsOfURL:fileURL];
       if (serializedLog && serializedLog.length && serializedLog.length > 0) {
-        id<MSLog> item = [NSKeyedUnarchiver unarchiveObjectWithData:serializedLog];
+        id<MSLog> item;
+        NSException *exception;
+
+        // Deserialize the log.
+        @try {
+          #pragma clang diagnostic push
+          #pragma clang diagnostic ignored "-Wold-style-cast"
```

Comment:
`static_cast`

Reply:
Aclually, I'd like to disable this useless (in our case) warning, but it's a different story.